libandroid/lib/src/main/java/com/mapbox/services/android/geocoder/ui/GeocoderAdapter.java

```diff
@@ -225,6 +230,20 @@ public void setLimit(int limit) {
     this.limit = limit;
   }
 
+  /**
+   * Can be used to cancel any calls currently in progress. It's a good idea to include in onDestroy() to prevent
+   * memory leaks
+   *
+   * @since 2.0.0
+   */
+  public void cancelApiCall() {
+    if (call == null) {
```

Comment:
Don't think we need to log this as it's not an uncommon occurrence. At most, log at debug I think.